### ISSUE TITLE
scss/operator-no-unspaced examines unquoted urls

### ISSUE THREAD

**Issue by srawlins** [kristerkari/stylelint-scss#452]
This test, added to `src/rules/operator-no-unspaced/__tests__/index.js` under `accept:`, at line 1263, fails:

```js
    {
      code: `
      @import url(//fonts.googleapis.com/css?family=Google+Material+Icons);
      `,
      description: "ignores unquoted urls"
    }
```

It complains about the `/` and `+` characters as operators.

**Comment by srawlins** [comment]
This one is confusing me because sometimes an unquoted `url()` in an `@import` is parsed as an expression:

```scss
/* url(10px + 2) */
@import url(10px + 2);

/* url(10px +2) */
@import url(10px +2);

/* url(10px+ 2) */
@import url(10px+ 2);

/* url(10px+2) */
@import url(10px+2);
```

is compiled by Sass as:

```css
/* url(10px + 2) */
@import url(12px);
/* url(10px +2) */
@import url(12px);
/* url(10px+ 2) */
@import url(12px);
/* url(10px+2) */
@import url(10px+2);
```

I've looked at the Sass docs, under the [url special function](https://sass-lang.com/documentation/syntax/special-functions#url), the [@import at-rule](https://sass-lang.com/documentation/at-rules/import), and quoted and unquoted [strings](https://sass-lang.com/documentation/values/strings).

I think this hinges on this bit from the url() docs:

> If the `url()`’s argument is a valid unquoted URL, Sass parses it as-is, although interpolation may also be used to inject SassScript values. If it’s not a valid unquoted URL—for example, if it contains variables or function calls—it’s parsed as a normal plain CSS function call.

@nex3 may be able to clarify:

What makes a "valid" unquoted URL? "Valid" as in, the URL spec?

**Comment by nex3** [comment]
It's "valid" according to [the CSS syntax for a `url()` production](https://drafts.csswg.org/css-syntax-3/#url-token-diagram).

**Comment by kristerkari** [comment]
This is now fixed in v3.15.0
https://github.com/kristerkari/stylelint-scss/releases/tag/3.15.0

Thanks everyone for helping out!
